src/test/java/lotto/domain/ResultTest.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ResultTest {
+    @DisplayName("매치 카운트랑 보너스 카운트에 맞는 합당한 결과를 알려 주는지")
+    @Test
+    void getResult() {
+        int matchCount = 0;
+        boolean bonusMatch = true;
+
+        assertThat(Result.getResult(matchCount, bonusMatch)).isEqualTo(Result.NONE);
+    }
+
+    @DisplayName("결과 값을 통계 리스트로 반환")
+    @Test
+    void resultStatistics() {
+        List<Integer> winningNumbers = Arrays.asList(1, 2, 3, 4, 5, 6);
+        int bonusNumber = 20;
+
+        Lotto lotto1 = new Lotto(Arrays.asList(1, 2, 3, 20, 21, 40)); // FIFTH
+        Lotto lotto2 = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 20)); // SECOND
+        Lottos lottos = new Lottos(Arrays.asList(lotto1, lotto2));
+
+        List<Result> results = lottos.getResults(winningNumbers, bonusNumber);
+        List<Integer> stats = Result.getStatistics(results);
+        assertThat(stats).isEqualTo(Arrays.asList(1, 0, 0, 1, 0));
+    }
```

Comment:
Result의 getStatistics() 메서드를 테스트하는 거라면 아래처럼 간단하게 진행하는건 어떨까요?
```suggestion
    void resultStatistics() {
        List<Integer> stats = Result.getStatistics(Arrays.asList(Result.SECOND, Result.FIFTH));
        assertThat(stats).isEqualTo(Arrays.asList(1, 0, 0, 1, 0));
    }
```